_downloads/federal-revenue-by-location.md

```diff
@@ -83,22 +83,22 @@ The offshore dataset is organized by offshore planning areas. There are more off
 
 ### Fields and definitions
 
-_Revenue Type._ Revenues from U.S. natural resources fall into one of several types:
+_Revenue Type_ Revenues from U.S. natural resources fall into one of several types:
```

Comment:
Should there be some sort of delimiter here? `_Revenue Type_.` perhaps?

Reply:
Hi @gemfarmer! Happy holidays!! I think we decided that the style difference is a sufficient delimiter here, but a colon might be a good alternative to periods.